src/town_voronoi.cpp

```diff
@@ -0,0 +1,378 @@
+/* $Id$ */
+
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file town_voronoi.cpp Handling of town Voronoi diagram, containing the TownID of the closest town for every tile. */
+
+#include "stdafx.h"
+#include "town.h"
+#include "sortlist_type.h"
+
+bool _voronoi_initialized;
+bool _hold_next_add;
+TownID _town_on_hold;
+
+typedef GUIList<const Town*> TownList;
```

Comment:
Could this not be a std::vector? I think the plan is to scrap our custom implementations and use std classes where we can.

Reply:
Done. (I'll just have to remove the shortlist include part.)